docs/pages/includes/desktop-access/desktop-config.yaml

```diff
@@ -24,6 +24,19 @@ windows_desktop_service:
     # likely "EXAMPLE". When connecting as the "svc-teleport" user, you should
     # use the format: "EXAMPLE\svc-teleport".
     username: '$LDAP_USERNAME'
+    # The security identifier of the service account specified by the username
+    # field above. This looks like a string starting with "S-".
+    #
+    # An administrator with access to the domain controller can obtain this value
```

Comment:
Do you actually need to be an administrator with access to a DC?

Reply:
From my research it seems like no. https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/plan/security-best-practices/appendix-b--privileged-accounts-and-groups-in-active-directory#permissions. I believe any user besides a guest. I will change it in an upcoming PR to say

"Any AD user with the permission to read user objects can obtain this value..."

Reply:
Fixed in https://github.com/gravitational/teleport/pull/19914